cloudmarker/stores/mongodbstore.py

```diff
@@ -0,0 +1,100 @@
+"""MongoDB store plugin."""
+
+
+import logging
+
+from pymongo import MongoClient
+
+_log = logging.getLogger(__name__)
+
+
+class MongoDBStore:
+    """A plugin to store records on the MongoDB."""
+
+    def __init__(self, host, port, username=None, password=None, db='gcp',
+                 buffer_size=1000):
+        """Initialize object of this class with specified parameters.
+
+        Arguments:
+            host (str): MongoDB instance host.
+            port (int): MongoDB instance port.
+            username (str): MongoDB instance auth username.
+            password (str): MongoDB instance auth password.
+            db (str): MongoDB database name, default: ``gcp``
+            buffer_size (int): Max number of records to hold in memeory for
+                each record_type.
+
+        """
+        # pylint: disable=too-many-instance-attributes
+
```

Comment:
mongodb db connection params: `self._mongodb_host`, `self._mongodb_port`, `self._mongodb_username`, `self._mongodb_password` are not being used anywhere else and are not required, they can be removed from the instance variables.